src/components/Hacktoberfest/Hacktoberfest.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react'
+import logo from './logo.svg'
+
+import "./Hacktoberfest.css"
+
+function Stat({ title, value }) {
+  return (
+    <div className="stat">
+      <span className="stat__title">{title}</span>
+      <span className="stat__value">{value}</span>
+    </div>
+  )
+}
+
+export function Hacktoberfest() {
+  const [data, setData] = useState([])
+  useEffect(() => {
+    fetch('https://europe-west1-github-insights-247314.cloudfunctions.net/hacktoberfest')
+      .then(res => res.json())
+      .then(setData)
+  }, [])
+
+  console.log('data', data)
+  const numberOfParticipants = data.length
+  const numberOfPullRequests = data.reduce((acc, { user }) => acc + user.contributionsCollection.pullRequestContributions.totalCount, 0)
+  const completionRate = data
+    .map(({ user }) => user.contributionsCollection.pullRequestContributions.totalCount)
+    .filter(pullRequestsTotal => pullRequestsTotal > 3)
```

Comment:
You sure about the filter, I think its more:
```
.filter(pullRequestsTotal => pullRequestsTotal >= 4)
```

Reply:
It is the same but your proposition is more clear 👍

Reply:
4 PRs must be done

Reply:
That wasn't the same since completion rate moved to 56 😂